src/app/components/dynamic-table/dynamic-table.component.ts

```diff
@@ -51,19 +56,19 @@ export class DynamicTableComponent
         for (let i = 0; i < columnName.length; i++) {
             if (columnName.charAt(i) === columnName.charAt(i).toUpperCase()) {
                 newColumnName += " " + columnName.charAt(i);
-            } else newColumnName += columnName.charAt(i);
+            } else {
+                newColumnName += columnName.charAt(i);
+            }
         }
         newColumnName = newColumnName.toLocaleLowerCase();
 
         return AppValues.capitalizeFirstLetter(newColumnName);
     }
 
-    /* eslint-disable  @typescript-eslint/no-explicit-any */
-    public onSelectRow(row: any): void {
-        this.onSelectRowEmit.emit(row);
+    public onSelectRow(row: string): void {
+        this.selectRowEmit.emit(row);
     }
 
-    /* eslint-disable  @typescript-eslint/no-explicit-any */
     private renderTable(data: any[]): void {
```

Comment:
This should be of `TableSourceInterface` type

Reply:
Fixed